packages/salesforce-adapter/src/filters/workflow.ts

```diff
@@ -194,8 +194,10 @@ const filterCreator: FilterCreator = () => {
           }
           const innerInstances = await Promise.all(makeArray(workflowInst.value[fieldName])
             .map(async innerValue => {
-              innerValue[INSTANCE_FULL_NAME_FIELD] = fullApiName(await apiName(workflowInst),
-                innerValue[INSTANCE_FULL_NAME_FIELD])
+              innerValue[INSTANCE_FULL_NAME_FIELD] = fullApiName(
+                await apiName(workflowInst),
+                innerValue[INSTANCE_FULL_NAME_FIELD]
```

Comment:
Super super nit: The reason why I prefer the ',' anyway is for cleaner commits in the future. If I were to add/remove any parameter from this function I would only see in the diff the line which changed and not the paremeter before as well.
```suggestion
                innerValue[INSTANCE_FULL_NAME_FIELD],
```